definition_validator/validator.cc

```diff
@@ -283,6 +355,7 @@ class ValidateWalk {
         validateTStructNotGrandparent(ctx.state, sym);
         validateAbstract(ctx.state, sym);
         validateAbstract(ctx.state, singleton);
+        validateFinal(ctx.state, sym.data(ctx), classDef.get());
```

Comment:
I see we're using `const T *const` in a bunch of places. We hardly do this anywhere else in Sorbet. If you really want to make sure something is super const, would you be more interested in taking a `const unique_ptr<T>&`?